### PR TITLE
visual feedback for wrong answer, distinguish roots

### PR DESCRIPTION
**Pull request by bencbernstein**
visual feedback for wrong answer *
Distinguish roots typographically from definitions *
text cutoff root definition solution *

### CODE REVIEW

**Comment by bencbernstein** [comment]
the last step of hinting doesn't work now. it might have something to do with it not finding the value in the word root since they're uppercased now. maybe there's better way to decouple and display the roots separately without touching the answer checking logic for case

File: Apiary/Library/Palette.swift [hunk 1]
```diff
@@ -23,15 +23,15 @@ enum Palette {
   var color: UIColor {
     switch self {
     case .blue:
-      return UIColor(hex: 0x0000FF)
+      return UIColor(hex: 0x56CCF2)
```
Comment (OLI9292) [inline Apiary/Library/Palette.swift]: why change this and red?

File: Apiary/ViewModels/GameViewModel.swift [hunk 2]
```diff
@@ -85,24 +85,24 @@ class GameViewModel {
...
-  func optionGuessed(_ guess: String, alreadyInUse: [String]) {
+  func optionGuessed(_ guess: String, alreadyInUse: [String]) -> Bool {
     
     // Check guess if displaying letters
     if optionsAreLetters && currentMode == .letterAndRoot {
       if let answer = answer.value, let root = answer.roots.first, answer.correct(guess: guess) {
         options.value = self.store.randomRootsWithSameLetter(as: root, total: 6)
         glowAnswer.value = false
-        return
+        return false
```
Comment (OLI9292) [inline Apiary/ViewModels/GameViewModel.swift]: i think this should return true

**Comment by OLI9292** [comment]
ya it looks like some things aren't working here.  I can't progress in question either.  changing the title to uppercase is definitely one thing that's throwing it off, since our words all come in as lowercase and that's whats saved in the models.

i think the hinting animations are still a little jarring and its worth spending as much time on them as we need to.  like for ex i don't like how the root text gets hiked up when the definition gets displayed, feels like the definition should just have a slow anim. right below and be a smaller text size or lighter color

**Comment by OLI9292** [comment]
maybe we always show the definition as labels but `isHidden` until a giveDefinitions flag is set to true

File: Apiary/Library/Palette.swift (hunk 1 above)
Comment (bencbernstein) [inline Apiary/Library/Palette.swift]: those are the figma representations of these colors

**Comment by bencbernstein** [comment]
yeah that makes sense, thinking of the hinting animations as holistic. i say we get them and this working an then talk it through before implementing

**Comment by OLI9292** [comment]
sounds good it might start working if you change to this `return optionButtons.flatMap { removeAfterSpace($0.currentTitle).lowercased() }`

i think it's to do with the roots being uppercased

**Pull request merged**